charts/substra-backend/templates/_helpers.tpl

```diff
@@ -408,3 +399,47 @@ Retrieve AWS environment variable value
 {{- end -}}
 {{- $value -}}
 {{- end -}}
+
+{{/*
+Define database secret name
+*/}}
+{{- define "substra-backend.database.secretName" -}}
+    {{- if .Values.database.auth.existingSecretName -}}
+        {{- .Values.database.auth.existingSecretName }}
+    {{- else -}}
+        {{ include "substra.fullname" . }}-database
+    {{- end -}}
+{{- end -}}
+
+{{/*
+Define redis secret name
+*/}}
+{{- define "substra-backend.redis.secretName" -}}
+    {{- if .Values.redis.auth.existingSecretName -}}
+        {{- .Values.database.auth.existingSecretName }}
```

Comment:
same here

Reply:
also, the new redis value has not been documented

Reply:
The redis section has:
```yaml
## @skip redis
```